google_project/variables.tf

```
@@ -48,6 +48,12 @@ variable "extra_project_labels" {
   default     = {}
 }
 
+variable "risk_level" {
+  default     = "low"
```

```suggestion
  default     = ""
```
I think this should be an empty string (to allow for backward compatibility) or no default at all.